pkg/apis/core/validation/validation_test.go

```diff
@@ -19033,6 +19034,18 @@ func TestValidateTopologySpreadConstraints(t *testing.T) {
 				field.NotSupported(nodeTaintsField, &unknown, supportedPodTopologySpreadNodePolicies.List()),
 			},
 		},
+		{
+			name: "key in MatchLabelKeys isn't correctly defined.",
+			constraints: []core.TopologySpreadConstraint{
+				{
+					MaxSkew:           1,
+					TopologyKey:       "k8s.io/zone",
+					WhenUnsatisfiable: core.DoNotSchedule,
+					MatchLabelKeys:    []string{"/simple"},
+				},
+			},
+			wantFieldErrors: []*field.Error{field.Invalid(fieldPathMatchLabelKeys, "/simple", "prefix part must be non-empty")},
```

Comment:
`prefix part must be non-empty` this error msg is hard to understand at the first glance until I checked the source which split the string by '/'. 

you may want to add another test string, such as an empty string, I am fine with this as is though.